chainerx_cc/chainerx/context_testdata/CMakeLists.txt

```diff
@@ -15,3 +15,13 @@ set_target_properties(backend1.so
     PREFIX ""
     SUFFIX ""
     LIBRARY_OUTPUT_DIRECTORY "${CMAKE_CURRENT_BINARY_DIR}/backends")
+
+if(MSVC)
+    target_link_libraries(backend0.so
+        chainerx
+        chainerx_native)
+
+    target_link_libraries(backend1.so
+        chainerx
+        chainerx_native)
+endif()
```

Comment:
Missing end-of-line here too. Also it seems like you're appending `chainerx` a second time.

Reply:
fixed it
https://github.com/durswd/chainer/commit/8d3fc33d35bd5d7c391a961a3aa5cfd6e9e6a2eb